src/c++/perf_analyzer/load_worker.h

```diff
@@ -274,6 +322,9 @@ class LoadWorker : public IWorker {
   // shared bool reference? Used to pause execution of this thread
   bool& execute_;
 
+  // All of the Inference contexts for this worker
+  std::vector<std::shared_ptr<InferContext>> ctxs_;
```

Comment:
Can you elaborate on this change a bit more? It looks like ConcurrencyWorker, before this PR, already had a vector of contexts, and presumably used all of them.

But RequestRateWorker had only one context, and now has a vector, but uses only one of them?

Reply:
Exactly. They had code that was 99% the same, but one knew it only had one context, while the other had a vector. Instead, giving RRW a vector of size 1 so they can both utilize the same code.